tools/dependency/release_dates.py

```diff
@@ -40,32 +42,58 @@ def verify_and_print_latest_release(dep, repo, metadata_version, release_date):
         latest_release = repo.get_latest_release()
         if latest_release.created_at > release_date and latest_release.tag_name != metadata_version:
             print(
-                f'*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}>: '
-                f'{latest_release.tag_name}@<{latest_release.created_at}>')
-    except github.UnknownObjectException:
+                Fore.YELLOW
+                + f'*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}>: '
+                f'{latest_release.tag_name}@<{latest_release.created_at}>' + Style.RESET_ALL)
+    except github.GithubException as e:
         pass
 
 
 # Print GitHub release date, throw ReleaseDateError on mismatch with metadata release date.
 def verify_and_print_release_date(dep, github_release_date, metadata_release_date):
     mismatch = ''
     iso_release_date = format_utc_date(github_release_date)
-    print(f'{dep} has a GitHub release date {iso_release_date}')
+    print(Fore.GREEN + f'{dep} has a GitHub release date {iso_release_date}' + Style.RESET_ALL)
     if iso_release_date != metadata_release_date:
         raise ReleaseDateError(f'Mismatch with metadata release date of {metadata_release_date}')
 
 
 # Extract release date from GitHub API.
 def get_release_date(repo, metadata_version, github_release):
     if github_release.tagged:
-        tags = repo.get_tags()
-        for tag in tags:
-            if tag.name == github_release.version:
-                return tag.commit.commit.committer.date
+
+        try:
+            latest = repo.get_latest_release()
+        except github.GithubException as e:
```

Comment:
likewise here - it would be good to know why this is safe to ignore - if its due to http failure or somesuch then perhaps we should add a warning "unable to retrieve github info" or similar